presto-main/src/test/java/com/facebook/presto/sql/gen/TestPageFunctionCompiler.java

```diff
@@ -116,14 +117,17 @@ public void testGeneratedClassName()
     {
         PageFunctionCompiler functionCompiler = new PageFunctionCompiler(createTestMetadataManager(), 0);
 
+        String dateTimePattern = "YYYYMMdd_HHmmss_SSS";
```

Comment:
Let's declare the pattern here (i.e. `DateTimeFormatter.ofPattern("YYYYMMdd_HHmmss_SSS");`)